api/conf/conf.go

```diff
@@ -36,8 +36,8 @@ const (
 	EnvDEV   = "dev"
 	EnvLOCAL = "local"
 
-	confPath   = "/go/manager-api/conf.json"
-	schemaPath = "/go/manager-api/schema.json"
+	confJsonPath = "/go/manager-api/conf.json"
```

Comment:
Is this a hard-coded configuration file?

Reply:
No, could be overridden by conf file.

Reply:
Sorry for my typo, this path would be overridden by ENV.

https://github.com/nic-chen/incubator-apisix-dashboard/blob/go-lint/docs/deploy.md#note

![image](https://user-images.githubusercontent.com/2106987/97855550-d9d4b480-1d35-11eb-8451-47f9f9caee05.png)

Reply:
And @nic-chen  will resolve this in the future for easy to use

Reply:
hemm, anyway, we can not hard-coded in code base.

Reply:
@moonming 
it's a history issue, we have a plan to refactor the config part in `2.1`